frontend/src/component/playground/Playground/AdvancedPlaygroundResultsTable/AdvancedPlaygroundResultsTable.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { screen } from '@testing-library/react';
+import { render } from 'utils/testRenderer';
+import { UIProviderContainer } from 'component/providers/UIProvider/UIProviderContainer';
+import { AdvancedPlaygroundResultsTable } from './AdvancedPlaygroundResultsTable';
+
+test('should render advanced playground table', async () => {
+    render(
+        <UIProviderContainer>
+            <AdvancedPlaygroundResultsTable
+                loading={false}
+                features={[
+                    {
+                        name: 'Infinite',
+                        projectId: 'ChangeReqs',
+                        environments: {
+                            development: [
+                                {
```

Comment:
I'd extract a method `feature(name: string)` that is parametrized with data that matters to the test and hide the irrelevant details inside method. You can e.g have identical strategies in each feature because you don't test them

As a result of that when I read the test my eyes can focus on the data that matters and filter out irrelevant plumbing